dpctl/tensor/_permute_dims.py

```diff
@@ -0,0 +1,52 @@
+#                       Data Parallel Control (dpctl)
+#
+#  Copyright 2020-2021 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+from itertools import permutations
+
+import dpctl.tensor as dpt
+
+
+def permute_dims(X, axes):
+    """
+    permute_dims(X: usm_ndarray, axes: tuple or list) -> usm_ndarray
```

Comment:
Since your branch does not build documentation we do not see how this comes out in docs. Please create a branch in the main repo and push there to see the rendered docs,